test/package.spec.ts

```diff
@@ -43,12 +46,12 @@ describe('Package', () => {
     expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toEqual(['README.md']);
   });
 
-  it('[E2E] should install productive dependencies only', async () => {
+  it('should show warning messages when old dependencies are used', async () => {
     const projectDir = getCleanProjectDir(testOutputDir, 'productive-dependencies');
     fs.copySync(nestAppDir, projectDir, { recursive: true });
     await Package.run([projectDir]);
 
-    expect(fs.readdirSync(path.resolve(projectDir, 'deployment'))).toIncludeAllMembers(['package.json', 'package-lock.json', 'node_modules']);
-    expect(fs.readdirSync(path.resolve(projectDir, 'deployment', 'node_modules', '@nestjs'))).not.toContain('cli');
+    expect(recordWarning).toHaveBeenCalledWith('Old SAP Cloud SDK: @sap/cloud-sdk-core is detected.');
+    expect(getWarnings).toHaveBeenCalled();
```

Comment:
Why are you overwriting this test completely? Should it not be copied so the original test still functions? (assuming that we need the functionality in the CLI)

Reply:
This is just a test commit, the original test will remain at the end.